shared_model/backend/protobuf/commands/proto_create_role.hpp

```diff
@@ -58,8 +47,19 @@ namespace shared_model {
       // lazy
       template <typename Value>
       using Lazy = detail::LazyInitializer<Value>;
-      const iroha::protocol::CreateRole &create_role_;
-      const Lazy<PermissionsType> role_permissions_;
+
+      const iroha::protocol::CreateRole &create_role_{proto_->create_role()};
+
+      const Lazy<PermissionsType> role_permissions_{[this] {
+        return boost::accumulate(
```

Comment:
Why not `std`?

Reply:
I did not change the logic, only place for the lambda.